src/graph.jl

```diff
@@ -81,6 +94,15 @@ function Base.transpose(S::SparsityPatternCSC{T}) where {T}
     return SparsityPatternCSC{T}(n, m, B_colptr, B_rowval)
 end
 
+# copied from SparseArrays.jl
+function Base.getindex(S::SparsityPatternCSC, i0::Integer, i1::Integer)
```

Comment:
This is useful in symmetric colorings when we check the diagonal coefficients with `iszero(S[i, i])`

Reply:
Original code can be inspected with
```julia
julia> using SparseArrays; @edit spzeros(2, 2)[1, 2]
```